apps/cowswap-frontend/.env

```
@@ -148,3 +148,10 @@
 # CROWDIN_PERSONAL_TOKEN
 
 REACT_APP_COW_CDN=https://files.cow.fi
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)